### PR TITLE
UI : Change Pipeline tasks ui to switch tabs

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue:  https://github.com/open-metadata/OpenMetadata/issues/8513#issuecomment-1319634779
-  Change Pipeline tasks ui to switch tabs

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM!

**Comment by ShaileshParmar11** [comment]
merge main and fix sonar

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx [hunk 1]
```diff
@@ -852,6 +857,7 @@ const PipelineDetails = ({
               gutter={[0, 16]}>
               <Col span={24}>
                 <Description
+                  //   className="m--l-md"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:860]: nit

Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:860]: remove

File: openmetadata-ui/src/main/resources/ui/src/styles/components/radio.less [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    border-color: @primary-color-lite;
+    z-index: 0;
+
+    &:hover {
+      background-color: @primary-color-lite;
+      color: @primary-color;
+      border-color: @primary-color-lite;
+    }
+  }
+}
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/styles/components/radio.less:40]: is it intentional to override the antd class globally?

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx]: @chirag-madlani  done!

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx]: @ShaileshParmar11  done!

**Comment by ShaileshParmar11** [comment]
Resolve conflict

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**